src/components/views/settings/NotificationSound.tsx

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright 2019 - 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, {createRef} from "react";
+
+import {_t} from "../../../languageHandler";
+import AccessibleButton, {ButtonEvent} from "../elements/AccessibleButton";
+import {chromeFileInputFix} from "../../../utils/BrowserWorkarounds";
+import {SettingLevel} from "../../../settings/SettingLevel";
+import {logger} from "../../../../../matrix-js-sdk/src/logger";
```

Comment:
This isn't compiling in our CI, because it needs to be an absolute import:

```suggestion
import {logger} from "matrix-js-sdk/src/logger";
```